Exercise3/Main.cpp

```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+
+int main()
+{
+	std::cout << "Multiples of 10: " << std::endl;
+	int sum = 0;
+	for (int i = 1; i <= 100; i++)
+	{
+		if (i % 10 == 0)
+		{
+			std::cout << i << " ";
+		}
+		else
```

Comment:
Remember, in this exercise, you shouldn't use else statements. Maybe you could try a break statement somewhere?